packages/web/src/migrations/update-9-2-0/set-build-libs-from-source.spec.ts

```diff
@@ -19,7 +20,7 @@ describe('set buildLibsFromSource to true', () => {
   it(`should set buildLibsFromSource to true`, async () => {
     tree.create(
       'workspace.json',
-      JSON.stringify({
+      serializeJson({
```

Comment:
This can be changed to use `writeJson` as well.

Reply:
@FrozenPandaz The Nx repo has no `writeJson` method for the Angular Devkit Tree. I can introduce a `writeJsonToTree` method in `packages/workspace/src/utils/ast-utils.ts` if you want me to?